src/org/labkey/test/tests/SampleTypeLookupDisplayColumnTest.java

```diff
@@ -0,0 +1,184 @@
+package org.labkey.test.tests;
```

Comment:
Should go in a module-specific package.
```suggestion
package org.labkey.test.tests.experiment;
```